en/developer_guide/specific_backbone_view.rst

```diff
@@ -0,0 +1,40 @@
+Backbone specific view
+======================
+
+Context
+-------
+
+As all the Backoffice of Open Orchestra is based on Backbone view system, all the specific event
+linked to a view are linked to the view directly.
+
+In some cases, during the integration of the Open Orchestra Backoffice, the generic view used for the
+edition page will not suit your needs. There are some new behaviour to add to the form, such as a
+specific event when you edit a field.
+
+There are two simple solutions to avoid :
+
+ - Add some javascript in the form template
+ - Add a global listener to the javascript project
+
+Both of these solutions are to avoid because :
+
+ - They couple the template and the javascript
+ - The may be checked each time the DOM is modified
+
+To prevent these usages, Open Orchestra provides a way to modify the Backbone view used by default.
+
+Usage
+-----
+
+As all the CRUD are based on an ``entityType``, you will have the possibility to add a view for the
+``edit`` and ``add`` action on this ``entityType``.
```

Comment:
How to do if the entity type is Content, but the modification target is only one Content Type, news for instance ?